src/lspsrc/umlmac.42

```diff
@@ -5,7 +5,7 @@
 ;;;  ** (c) Copyright 1981 Massachusetts Institute of Technology *************
 ;;;  *************************************************************************
 
-(herald UMLMAC /40)
+(herald UMLMAC /42)
```

Comment:
Could just say `(herald UMLMAC)`, and it'll use the FN2 as the version (e.g. like MAXTUL does).

Reply:
You're probably right.  I was just following the pattern used for this file over the years. The version was always explicit.  And in this case, wrong.